BootloaderCorePkg/Library/PciEnumerationLib/InternalPciEnumerationLib.h

```diff
@@ -81,6 +81,13 @@ struct _PCI_BAR_RESOURCE {
   PCI_BAR                                  *PciBar;
 };
 
+typedef struct _PCI_BUS_NUM_RANGE          PCI_BUS_NUM_RANGE;
+
+struct _PCI_BUS_NUM_RANGE {
+  UINT8                                     BusBase;
+  UINT8                                     BusLimit;
+};

```

Comment:
It looks it is not necessary to define _PCI_BUS_NUM_RANGE, it would be simple as below:
typedef struct {
  UINT8                                     BusBase;
  UINT8                                     BusLimit;
} PCI_BUS_NUM_RANGE;

Reply:
Addressed.